src/wallet/test/wallet_tests.cpp

```diff
@@ -7,6 +7,7 @@
 #include <future>
 #include <memory>
 #include <stdint.h>
+#include <thread>
```

Comment:
nit: while touching adjacent line, could help a neighbour out
```suggestion
#include <cstdint>
#include <future>
#include <memory>
#include <thread>
```

Reply:
Done.